docs/examples.md

```diff
@@ -498,7 +498,7 @@ If you're the author of a third-party library with *attrs* integration, please s
 
 ## Types
 
-*attrs* also allows you to associate a type with an attribute using either the *type* argument to {func}`attr.ib` and {func}`attr.field` or using {pep}`526`-annotations:
+*attrs* also allows you to associate a type with an attribute using either the *type* argument to using {pep}`526`-annotations or {func}`attrs.ib` and {func}`attrs.field`:
```

Comment:
```suggestion
*attrs* also allows you to associate a type with an attribute using either the *type* argument to using {pep}`526`-annotations or {func}`attrs.field`/{func}`attr.ib`:
```

We can put field first, but we shouldn't erase ib completely, since it helps people to navigate older attrs code bases. Also attrs.ib doesn't exist. :)